addons/website/views/website_views.xml

```diff
@@ -138,6 +139,7 @@
                     <field name="is_mega_menu"/>
                     <field name="new_window"/>
                     <field name="parent_id" context="{'display_website': True}"/>
+                    <field name="group_ids" widget="many2many_tags"/>
```

Comment:
Fun fact: you can block yourself out of being able to edit a menu. :grimacing: 
If you select a group you don't belong to, upon save you get an error, and if you then refresh or go to another page without correcting the problem, you can't come back to editing the menu.
The message is along the lines of "you do not have read access to this record, contact your administrator".
Shouldn't we either 1) make sure admins are always included by default as soon as there's at least one group registered, or 2) show an error preventing to save if you didn't put a group which includes yourself? (1 is probably more robust, because in 2 you could always remove the group afterwards and then nobody would have access to it.)